common/src/main/java/com/wynntils/wynn/utils/DiscoveryUtils.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wynn.utils;
```

Comment:
This does not really belong in `wynn.utils`. `openDiscoveryOnMap`, `setDiscoveryCompass` can be inlined into `DiscoveryManager`. I feel `locateSecretDiscovery` should preferably be in `WebManager`

Reply:
Maybe not wynn.utils, but imo definitely worth a util class

Reply:
I don't see much reuse of this code, unless you are planning something akin to `/discovery` In that case, I still feel `locateSecretDiscovery` should be moved